cluster/up.sh

```diff
@@ -19,43 +19,47 @@ set -ex
 source ./cluster/kubevirtci.sh
 CNAO_VERSIOV=0.35.0
 KUBEVIRT_VERSION=v0.29.0
+DEPLOY_KUBEVIRT=${DEPLOY_KUBEVIRT:-true}
 kubevirtci::install
 
-if [[ "$KUBEVIRT_PROVIDER" != external ]]; then
-    $(kubevirtci::path)/cluster-up/up.sh
-fi
+$(kubevirtci::path)/cluster-up/up.sh
 
-# Deploy CNA
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/namespace.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/network-addons-config.crd.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/operator.yaml
-./cluster/kubectl.sh create -f ./hack/cna/cna-cr.yaml
+if [[ "${KUBEVIRT_PROVIDER}" != external ]]; then
+    $(kubevirtci::path)/cluster-up/up.sh
 
-# wait for cluster operator
-./cluster/kubectl.sh wait networkaddonsconfig cluster --for condition=Available --timeout=800s
+    # Deploy CNAO
```

Comment:
Do we want to assume that external cluster should have CNAO deployment?

Reply:
mmm... that's a valid question. I assumed so, I not really necessary I guess. what do you propose?

Reply:
ok I discussed this with @phoracek and in case of external provider we should not deploy this, as it is the responsibility of the external cluster itself

Reply:
I believe that cluster-up should be no-op on external cluster. We should be only interested in cluster-sync.

Reply:
mmmm... see I thought we could use `make cluster-up` to deploy kubevirt on CNAO repo before running the tier 1 tests. 
Should we do this seperately then? @phoracek

Reply:
I would not want an external testing script to install something unexpected on my cluster. Explicitly installing KubeVirt on CNAO side would be preferable IMO. Or keeping a separate helper script on KMP, but executed explicitly by owner of the external cluster.

Reply:
ack, 
I want to avoid having to bump the KUBEVIRT_VERSION in both the CNAO teir1 test script & KMP repo, so I think a separate script on KMP is the best choise here.

Reply:
Or just always fetch the latest: https://github.com/kubevirt/kubevirt/releases/latest

Reply:
kubevirt_deploy script should be everywhere you want to deploy kubevirt, this must be automated and separated from the main cluster deployment, we can get kubevirt version as a parameter so it will be easier to maintain.
IMO there is no problem having kubevirt_deploy in kubemacpool and cnao these are different projects

Reply:
Well that won't help with downloading the artefact directly but I'm sure there is a way to curl your way there.